src/cnxml/deserialize.js

```diff
@@ -177,13 +177,14 @@ function emphasis(el) {
 function xref(el) {
     const target = el.getAttribute('target-id') || null
     const url = el.getAttribute('url') || null
+    const declension = el.getAttribute('cmlnle-case') || null
 
     if (target) {
         return {
             object: 'inline',
             type: 'xref',
             isVoid: true,
-            data: { target },
+            data: { target, declension },
```

Comment:
This should be called _case_; _declension_ in English is not the same as a _grammatical case_.

> **declension**
>
> 1. A falling off, decay or descent.
> 2. (grammar) The act of declining a word; the act of listing the inflections of a noun, pronoun or adjective in order.
> 3. (grammar) A way of categorizing nouns, pronouns, or adjectives according to the inflections they receive. 
>
> (from [Wiktionary](https://en.wiktionary.org/wiki/declension))

> **(grammatical) case**
>
> _[..]_
>
> 6. (grammar) A specific inflection of a word depending on its function in the sentence.
> 
> and
>
> 1. (grammar) A mode of inflection of a word dependent on its use, especially the syntactic function in a phrase. 
>
> (from [Wiktionary](https://en.wiktionary.org/wiki/case#Noun), [Wiktionary](https://en.wiktionary.org/wiki/grammatical_case#English))